src/cmd-build

```diff
@@ -216,31 +220,43 @@ fi
 
 /usr/lib/coreos-assembler/gf-anaconda-cleanup "$(pwd)"/"${img_base}"
 /usr/lib/coreos-assembler/gf-oemid "$(pwd)"/"${img_base}" "$(pwd)"/"${img_qemu}" qemu
-set +x
 # make a version-less symlink to have a stable path
 ln -s "${img_qemu}" "${name}"-qemu.qcow2
 
 img_qemu_sha256=$(sha256sum "${img_qemu}" | cut -f 1 -d ' ')
-
+img_qemu_size=$(stat --format=%s "${img_qemu}")
 build_timestamp=$(date -u +$RFC3339)
+kickstart_checksum=$(awk '/SHA256.*iso/{print$NF}' "${workdir}"/installer/*CHECKSUM)
```

Comment:
Minor regression introduced in #299.

Reply:
Hmm...that checksum is supposed to just be the "image config" now.  It shouldn't IMO include the installer ISO.

Reply:
If we want to include that it should be a separate variable.

Reply:
It's been renamed. See the comment below about why I included it.

Reply:
OK, I see.